opentimelineio/adapters/fcp_xml.py

```diff
@@ -238,11 +248,11 @@ def _parse_clip_item_without_media(clip_item, track_rate,
     ]
 
     in_frame = (
-        int(clip_item.find('./in').text)
+        int(float(clip_item.find('./in').text))
```

Comment:
Why `int(float(x))`? Would you ever want to keep the fractional part? Also in the line below there's an `int(round(x))` Is the rounding important? It seems odd that they don't match.

Reply:
Hi again!
I did some additional testing and to avoid `ValueError` when receiving `'0.0'` from `clip_item.find('./in').text` it needs to be converted to a float first. Same behavior in python 2 and 3.
```
>>> int('0.0')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: '0.0'
>>> 
```
Found someone else stumbling across the same dilemma: https://stackoverflow.com/questions/17142088/python-2-7-and-3-3-2-why-int0-0-does-not-work